Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UITransform/UITransformUpdateSystem.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.Throttling;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.SDKComponents.SceneUI.Components;
+using DCL.SDKComponents.SceneUI.Utils;
+using ECS.Abstract;
+using ECS.Groups;
+using SceneRunner.Scene;
+using UnityEngine.UIElements;
+
+namespace DCL.SDKComponents.SceneUI.Systems.UITransform
+{
+    [UpdateInGroup(typeof(SyncedSimulationSystemGroup))]
+    [UpdateAfter(typeof(UITransformSortingSystem))]
+    [LogCategory(ReportCategory.SCENE_UI)]
+    [ThrottlingEnabled]
+    public partial class UITransformUpdateSystem : BaseUnityLoopSystem
+    {
+        private readonly UIDocument canvas;
+        private readonly ISceneStateProvider sceneStateProvider;
+
+        public UITransformUpdateSystem(World world, UIDocument canvas, ISceneStateProvider sceneStateProvider) : base(world)
+        {
+            this.canvas = canvas;
+            this.sceneStateProvider = sceneStateProvider;
+        }
+
+        protected override void Update(float _)
+        {
+            UpdateUITransformQuery(World);
+            CheckUITransformOutOfSceneQuery(World);
```

Comment:
It's not necessary to call this query every frame. It will lead to extra overhead that can be avoided:
this query should be executed only if `IsCurrent` state has changed: you can store it in the system itself and check. 
Thus, the overhead will be negligible.

Reply:
You're right. I've added a `lastIsCurrentScene` variable to check it and now it works as expected.

Done here: https://github.com/decentraland/unity-explorer/pull/204/commits/8d87f3ad96b37c47d0799b1269d20cea35dc3090